Wrong shot number:
[10, 11, 16]

Correct shot number:
[15]

Raw json:

{'hasError': True, 'errParameter': ['hadoop.security.group.mapping.ldap.connection.timeout.ms'], 'reason': ["The property 'hadoop.security.group.mapping.ldap.connection.timeout.ms' has the value '120000' which is out of the valid range of 0 to 1000000000."]}

{'hasError': True, 'errParameter': ['hadoop.security.group.mapping.ldap.connection.timeout.ms'], 'reason': ["The property 'hadoop.security.group.mapping.ldap.connection.timeout.ms' has the value '120000' which is out of the valid range of 0 to 1000000000."]}

{'hasError': True, 'errParameter': ['hadoop.security.group.mapping.ldap.connection.timeout.ms'], 'reason': ["The property 'hadoop.security.group.mapping.ldap.connection.timeout.ms' has the value '120000' which is not within the accepted range of 0 to 1000000000."]}

Final result:

There are 1 misconfiguration parameters in the input: hadoop.security.group.mapping.ldap.connection.timeout.ms
[Ciri] Reason for hadoop.security.group.mapping.ldap.connection.timeout.ms: The property 'hadoop.security.group.mapping.ldap.connection.timeout.ms' has the value '120000' which is not within the accepted range of 0 to 1000000000.
